src/audio/host.c

```diff
@@ -404,13 +404,11 @@ static int host_copy_normal(struct comp_dev *dev)
 
 	copy_bytes = host_get_copy_bytes_normal(dev);
 	if (!copy_bytes)
-		return ret;
```

Comment:
Hmm I like to return "ret" more TBH this way we are consistent of always returning the same variable and not magic numbers.

Reply:
I usually approach this firstly from the PoV of minimalism, secondly of locality. You don't do
```
int f(void)
{
	int ret = 0;
	return ret;
}
```
you just do
```
int f(void)
{
	return 0;
}
```
I think this is similar. Locality - when I look at that line I immediately know what happens there. With the previous version I have to check 9 lines up to see where `ret` comes from and whether it was modified in between. So, sorry, no, I prefer this version.

Reply:
Both approaches can be considered better or worse... I think we should create a macro or enum which maps 0 with some meaningful success constant like in CAVS we have ADSP_SUCCESS. Anyway, if you feel like returning 0 instead of defined ret is more appropriate then OK, this is not critically important tbh.